interface/weno/adminClass.php

```diff
@@ -0,0 +1,161 @@
+                        `zipcode` int(5) DEFAULT NULL,
+                        `retail` varchar(4) DEFAULT NULL,
+                        `specialty` varchar(5) DEFAULT NULL,
+                        `long_term_care` varchar(5) DEFAULT NULL,
+                        `Mail_Order` varchar(5) DEFAULT NULL,
+                        `Mail_State_Codes` varchar(3) DEFAULT NULL,
+                        `Mail_Address_Line_1` varchar(35) DEFAULT NULL,
+                        `Mail_Address_Line_2` varchar(16) DEFAULT NULL,
+                        `Mail_City` varchar(10) DEFAULT NULL,
+                        `Mail_State` varchar(2) DEFAULT NULL,
+                        `Mail_Zip_Code` int(9) DEFAULT NULL,
+                        `Mail_Phone` int(10) DEFAULT NULL,
+                        `Mail_Fax` int(10) DEFAULT NULL,
+                        `EPCS_Permitted` varchar(5) DEFAULT NULL,
+                        `Accept_NewRx` varchar(4) DEFAULT NULL,
+                        `Accept_Refillresponse` varchar(5) DEFAULT NULL,
+                        `Accept_RxChangeresponse` varchar(5) DEFAULT NULL,
+                        `Accept_Verify` varchar(5) DEFAULT NULL,
+                        `Accept_Cancelrx` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryrequest` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryresponse` varchar(5) DEFAULT NULL,
+                        `Accept_Census` varchar(5) DEFAULT NULL,
+                        `Accept_Resupply` varchar(5) DEFAULT NULL,
+                        PRIMARY KEY (`id`)
+                        ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=11150"
+    		);
+
+          sqlStatement(
+            "CREATE TABLE IF NOT EXISTS `erx_rx_log` (
+                         `id` int(20) NOT NULL AUTO_INCREMENT,
+                         `prescription_id` int(6) NOT NULL,
+                         `date` varchar(25) NOT NULL,
+                         `time` varchar(15) NOT NULL,
+                         `code` int(6) NOT NULL,
+                         `status` text NOT NULL,
+                         `message_id` varchar(100) DEFAULT NULL,
+                         `read` int(1) DEFAULT NULL,
+                         PRIMARY KEY (`id`)
+                          ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=133" 
+          	);
+
+          sqlStatement(
+              "CREATE TABLE IF NOT EXISTS `erx_erx_narcotics` (
+                            `id` int(11) NOT NULL AUTO_INCREMENT,
+                            `drug` varchar(255) NOT NULL,
+                            `dea_number` varchar(5) NOT NULL,
+                            `csa_sch` varchar(2) NOT NULL,
+                            `narc` varchar(2) NOT NULL,
+                            `other_names` varchar(255) NOT NULL,
+                             PRIMARY KEY (`id`)
+                           ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=570"
+          	);
+
```

Comment:
Instead add these tables to the standard database sql file and the standard sql upgrade file. It's only 4 tables and makes way more sense (allows easy modification/improvement of tables if they are always there). And this will turn into a very important feature since plan to use for MU3.